allennlp/training/metrics/fbeta_multi_label_measure.py

```diff
@@ -0,0 +1,210 @@
+
+    # Parameters
+
+    beta : `float`, optional (default = `1.0`)
+        The strength of recall versus precision in the F-score.
+
+    average : `str`, optional (default = `None`)
+        If `None`, the scores for each class are returned. Otherwise, this
+        determines the type of averaging performed on the data:
+
+        `'micro'`:
+            Calculate metrics globally by counting the total true positives,
+            false negatives and false positives.
+        `'macro'`:
+            Calculate metrics for each label, and find their unweighted mean.
+            This does not take label imbalance into account.
+        `'weighted'`:
+            Calculate metrics for each label, and find their average weighted
+            by support (the number of true instances for each label). This
+            alters 'macro' to account for label imbalance; it can result in an
+            F-score that is not between precision and recall.
+
+    labels: `list`, optional
+        The set of labels to include and their order if `average is None`.
+        Labels present in the data can be excluded, for example to calculate a
+        multi-class average ignoring a majority negative class. Labels not present
+        in the data will result in 0 components in a macro or weighted average.
+
+    threshold: `float`, optional (default = `0.5`)
+        Logits over this threshold will be considered predictions for the corresponding class.
+
+    """
+
+    def __init__(
+        self,
+        beta: float = 1.0,
+        average: str = None,
+        labels: List[int] = None,
+        threshold: float = 0.5,
+    ) -> None:
+        average_options = {None, "micro", "macro", "weighted"}
+        if average not in average_options:
+            raise ConfigurationError(f"`average` has to be one of {average_options}.")
+        if beta <= 0:
+            raise ConfigurationError("`beta` should be >0 in the F-beta score.")
+        if labels is not None and len(labels) == 0:
+            raise ConfigurationError("`labels` cannot be an empty list.")
+        self._beta = beta
+        self._average = average
+        self._labels = labels
+        self._threshold = threshold
+
+        # statistics
+        # the total number of true positive instances under each class
+        # Shape: (num_classes, )
+        self._true_positive_sum: Union[None, torch.Tensor] = None
+        # the total number of instances
+        # Shape: (num_classes, )
+        self._total_sum: Union[None, torch.Tensor] = None
+        # the total number of instances under each _predicted_ class,
+        # including true positives and false positives
+        # Shape: (num_classes, )
+        self._pred_sum: Union[None, torch.Tensor] = None
+        # the total number of instances under each _true_ class,
+        # including true positives and false negatives
+        # Shape: (num_classes, )
+        self._true_sum: Union[None, torch.Tensor] = None
```

Comment:
```suggestion
        super().__init__(beta, average, labels)
        self._threshold = threshold
```
Any reason we can't do this?

Reply:
Much simpler, good catch! Fixed.